(env) ➜  aws-priceing git:(main) ✗ pythoh app.py
zsh: command not found: pythoh
(env) ➜  aws-priceing git:(main) ✗ python app.py
 * Serving Flask app 'app'
 * Debug mode: on
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://10.120.122.140:8080
Press CTRL+C to quit
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 113-789-396
10.120.122.140 - - [05/May/2025 13:52:28] "GET / HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/srini/env/lib/python3.8/site-packages/flask/app.py", line 1498, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/srini/env/lib/python3.8/site-packages/flask/app.py", line 1476, in wsgi_app
    response = self.handle_exception(e)
  File "/home/srini/env/lib/python3.8/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/srini/env/lib/python3.8/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/srini/env/lib/python3.8/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/srini/env/lib/python3.8/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/srini/aws-priceing/app.py", line 129, in home
    return render_template_string(FORM_PAGE, css=CSS, js=SORT_JS, template_folder='.', globals={'base':BASE_PAGE})
  File "/home/srini/env/lib/python3.8/site-packages/flask/templating.py", line 162, in render_template_string
    return _render(app, template, context)
  File "/home/srini/env/lib/python3.8/site-packages/flask/templating.py", line 131, in _render
    rv = template.render(context)
  File "/home/srini/env/lib/python3.8/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/home/srini/env/lib/python3.8/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 2, in top-level template code
  File "/home/srini/env/lib/python3.8/site-packages/flask/templating.py", line 65, in get_source
    return self._get_source_fast(environment, template)
  File "/home/srini/env/lib/python3.8/site-packages/flask/templating.py", line 99, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: base
